### DISCUSSION TITLE
AI Tool ignores Telegram limit size percentages that I set in camera triggering objects.

### DISCUSSION THREAD

**Discussion by arthmost** [VorlonCD/bi-aidetection#329] [Q&A] [answered]
Hi,
Can someone help?
I don't know what to do else, because AI Tool ignores everything that I set up for Telegram notifications.
For example:
In **Relevant Objects** cameraname\Default I set **Limit size percentage** for Person: 0%-100%
Then in **Actions** I followed **Triggering Objects** and set up for cameraname\Telegram next **Limit size percentage** for Person: 1%-15%
So, say, if a person is less than 1% or bigger than 15% I shouldn't get a telegram notification. But I get it every time.
The same thing with a **Object Confident Limits**. 
The settings just don't work and don't affect anything.

How to fix that? Maybe I got wrong these settings and it should work that way, I don't know. I want to detect objects, but send notifications if it meets the specified parameters.
Everything except this works flawless.

The screenshots: https://imgur.com/a/xKVRPHv
The test log file: https://drive.google.com/file/d/1EhVaLUq70EdZ7bQMSyeMiixlTURmAaXp/view?usp=share_link

Version: 2.2.24.8133

**Comment by VorlonCD** [comment, marked as answer]
Your log shows this:
`RelevantObjectManager: Allowing 'Telegram' because no relevant objects were enabled. 1 predictions(s), Enabled=0 of 28`

I'm not sure why 0 are enabled, your screenshots seem to show them enabled.

Try this:

1. If you dont have a camera called "default", then create it.   This is used for several things including when you hit RESET in relevant objects manager.
2. If you already HAD DEFAULT camera, edit that cameras Relevant objects
3. Hit RESET
4. Make sure all the defaults are what makes sense for most cameras.
5. Go to the actual camera you are having trouble with > Actions > Telegram Triggering Objects
6. Hit RESET
7. Now tweak your size percentage, etc.
8. Test again

**Reply by VorlonCD** [reply]
@arthmost  @nefh - just to confirm, this fixed it?

**Reply by arthmost** [reply]
> @arthmost @nefh - just to confirm, this fixed it?

Yeah, fixed! Thank you, it works perfectly!
